.circleci/config.yml

```diff
@@ -50,7 +50,7 @@ jobs:
 
 # code analysis
   test-static-code-analysis:
-    resource_class: medium+
+    resource_class: xlarge
```

Comment:
Is resource class `large`  solves the problem?

Reply:
Ok